.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug report
+description: Create a bug report to help us improve. Use this template if you encountered an issue while integrating with or implementing the APIs of this SDK.
+labels: [bug, triage-required]
+
+body:
+- type: checkboxes
+  attributes:
+    label:  Before logging a bug report please confirm the following
+    description: Go through the items below before logging an issue.
+    options:
+    - label: This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-edge-ios/security/policy).
+      required: true
+    - label: I have searched in this repository's issues to see if it has already been reported.
+      required: true
+    - label: I have updated to the latest released version of the SDK and the issue still persists.
+      required: true
+
+- type: textarea
+  attributes:
+    label: Bug summary
+    description: Please provide a summary of the bug you are reporting.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      Please provide the OS version, SDK version(s) used, IDE version, and any other specific settings that could help us narrow down the problem.
+      Example:
+        - **OS**: iOS 15.5
+        - **SDK(s)**: AEPCore 3.7.0
+        - **IDE**: Xcode 13.4
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Steps to reproduce
+    description: Steps to reproduce the behavior consistently.
+    placeholder: |
+      1. ...
+      2. ...
+      3. ...
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: Current behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: false

```

Comment:
Were the current and expected behavior fields marked non mandatory because they are generally included in the description?  IMO, for a bug report, it is worthwhile making them required as it will provide more clarity on what the reporter expectation is.

Reply:
Currently in the other extensions the issue forms we have mark some or all these fields as required but are not really enforced by in the UI.  If we mark them all as required might be too restrictive, I thought we can start by enforcing the most important ones and we can elevate some of the other sections as required as we see fit over time. what do you think?

Reply:
Sounds good! I am aligned with making them required as needed.